external/opensocdebug/hardware/modules/system_diagnosis/common/source/diagnosis_missing_defines.vh

```diff
@@ -0,0 +1,7 @@
+// Address of the Diagnosis Processor
```

Comment:
File deleted. Defines were used for the old packetizer "packetizer_dgbnoc_if.v" and for the old address of the debug processor. File isn't needed anymore because a new packetizer is used and the address of the debug processor is set to the constant value 6 in Packetizer.v